tests/gate_data.py

```diff
@@ -438,3 +438,37 @@ def DoubleExcitationMinus(phi):
     U[12, 12] = c
 
     return U
+
+
+def QuantumNumberPreservingOR(theta):
```

Comment:
The name of the parameter for this gate is different than the one used in the figure.

Reply:
Fixed.